airbyte-integrations/connectors/source-airtable/source_airtable/streams.py

```diff
@@ -71,7 +71,11 @@ def parse_response(self, response: requests.Response, **kwargs) -> Mapping[str,
             }
         """
         records = response.json().get(self.name)
-        yield from records
+        for base in records:
+            if base.get("permissionLevel") == "none":
+                self.logger.warning(f"Skipping base `{base.get('name')}` with id `{base.get('id')}`: Not sufficient permissions")
```

Comment:
Not > Not enough permissions. 

+ add the link to the docs where this could be observed and applied in Airtable UI.

Reply:
updated with direct message depending on type of authentication is used